types/chai/index.d.ts

```diff
@@ -330,7 +331,7 @@ declare namespace Chai {
     }
 
     interface OwnProperty {
-        (name: string, message?: string): Assertion;
+        (name: string, value?: any, message?: string): Assertion;
```

Comment:
Shouldn't this be an overload instead of replacing the existing signature?

Reply:
I basically made it the same as Property interface. I'm not sure what reads better 2 optional parameters or 2 overloaded methods with optional parameters.
I'm good with adding overloaded method. What do you think of adding the method to Property interface and delete OwnProperty interface?

Reply:
Technically, what you've done here will still work, but the overload is more expressive as to what the middle argument represents in the case that you only provide 2 arguments. That's why I would suggest the overload. If the `Property` interface works the same, perhaps it should have an overload too.

Reply:
I made adjustments to the `Property` interface and deleted `OwnProperty` interface.